src/ppx/graphics_util.cpp

```diff
@@ -966,6 +966,85 @@ Result CreateTextureFromBitmap(
     return ppx::SUCCESS;
 }
 
+Result CreateTextureFromMipmap(
+    grfx::Queue*          pQueue,
+    const Mipmap*         pMipmap,
+    grfx::Texture**       ppTexture,
+    const TextureOptions& options)
+{
+    PPX_ASSERT_NULL_ARG(pQueue);
```

Comment:
Not for this CL, but in general, I wonder if this should be PPX_ASSERT_NOT_NULL_ARG

Reply:
I don't have a strong opinion on this. The original logic was is assert on the condition in the macro name, .e.g. assert on null arg. This maybe different than how others think about the assert if expression inside assert() is false.

Reply:
Thanks for the explanation! I usually read this as "assert (that) condition/assumption", and I think that's how it's usually defined in other places. Just by looking at the function name I had to look at the definition to see what it was meant to do. It'd be different if it was named THROW_IF_COND, FAIL_IF_COND. Anyway, it's not related to this PR, so not really important.